CHANGELOG.md

```diff
@@ -18,6 +18,23 @@ Given a version number MAJOR.MINOR.PATCH:
 
 ---
 
+## [1.3.0](https://github.com/freiheit-com/kuberpult/tree/1.3.0) (2023-09-05)
+
+[Full Changelog](https://github.com/freiheit-com/kuberpult/compare/1.2.0...1.3.0)
+
+### Minor Changes
+
+- Fix layout in Lock tooltip [\#913](https://github.com/freiheit-com/kuberpult/pull/913) ([sven-urbanski-freiheit-com](https://github.com/sven-urbanski-freiheit-com))
+
+**Merged pull requests:**
```

Comment:
this sections "Merged pull requests" shouldn't exist.

It seems that the PRs are not properly labeled.

Reply:
... but they are labelled.
Not sure why this happened, but please sort them under "Minor Changes"

Reply:
Done